migration/src/jira2github_import.py

```diff
@@ -74,6 +75,14 @@ def convert_issue(num: int, dump_dir: Path, output_dir: Path, account_map: dict[
             attachment_list_items.append(f"[{filename}]({attachment_url(num, filename, att_repo, att_branch)})" + (f" (versions: {cnt})" if cnt > 1 else ""))
             att_replace_map[filename] = attachment_url(num, filename, att_repo, att_branch)
 
+        # detect unmentioned image files
+        # https://github.com/apache/lucene-jira-archive/issues/126
+        image_files = [x[0] for x in attachments if re.match(r"^.+\.(png|jpg|jpeg|gif|svg)$", x[0], flags=re.IGNORECASE)]
```

Comment:
I thought about `.tiff`, `.tif`, `.bmp` too, but I grep'd my local `jira-dump` and found no such attachments.  Maybe Jira does not support them anyways.

Reply:
I added other image file types in https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types.
https://github.com/apache/lucene-jira-archive/pull/135/commits/c858bc4509f1e2651f44073e101a51cb94ac0f4f